### PR TITLE
chore(check-for-update): add some logging for readability

### PR DESCRIPTION
**Pull request by janpio**
Output is currently hard to parse visually, so added some empty lines and separators to hopefully make it easier to read.

### CODE REVIEW

File: .github/scripts/check-for-update.sh [hunk 1]
```diff
@@ -47,25 +49,33 @@ minutes=5   # cron job runs each x minutes
...
-
-  echo "checking info..."
 
+  echo "=========================="
+  echo "getting package version:"
   v=$(bash .github/scripts/prisma-version.sh "$branch")
-
+  echo "$v (via Npm)"
+  
+  packages=$(find . -not -path "*/node_modules/*" -type f -name "package.json")
```
Comment (janpio) [inline .github/scripts/check-for-update.sh:76]: moved from above

File: .github/scripts/check-for-update.sh [hunk 2]
```diff
@@ -47,25 +49,33 @@ minutes=5   # cron job runs each x minutes
...
   git reset --hard "github/$branch"
-  packages=$(find . -not -path "*/node_modules/*" -type f -name "package.json")
-
-  echo "checking info..."
 
+  echo "=========================="
+  echo "getting package version:"
   v=$(bash .github/scripts/prisma-version.sh "$branch")
-
+  echo "$v (via Npm)"
```
Comment (janpio) [inline .github/scripts/check-for-update.sh:74]: new

File: .github/scripts/check-for-update.sh [hunk 3]
```diff
@@ -80,6 +90,7 @@ while [ $i -le $count ]; do
     hasResolutions="$(node -e "$pkg;console.log(!!pkg.resolutions)")"
 
     if [ "$hasResolutions" = "true" ]; then
+      echo "note: project uses `resolutions`"
```
Comment (janpio) [inline .github/scripts/check-for-update.sh:93]: new

File: .github/scripts/check-for-update.sh [hunk 4]
```diff
@@ -126,12 +137,16 @@ while [ $i -le $count ]; do
           echo "$item: @prisma/client expected $v, actual $vPrismaClient"
           yarn add "@prisma/client@$v"
         fi
+      else
+        echo "Dependency not found"
```
Comment (janpio) [inline .github/scripts/check-for-update.sh:141]: new

**Review by divyenduz — APPROVED** [review]
